.argo-ci/ci.yaml

```diff
@@ -30,6 +30,12 @@ spec:
             value: "{{item}}"
         withItems:
         - dep ensure && make cli lint test
```

Comment:
I think we can remove `test`  and keep only `cli` and `lint` since `test-coverage` step will execute unit tests either way.